content/blog/2017/2017-02-14-pipleine-editor-preview.adoc

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Say 'Hello' to the Blue Ocean Pipeline Editor"
+tags:
+- blueocean
+- editor
+- declarative
+- pipeline
+author: jamesdumay
+---
+
+link:/blog/2016/09/19/blueocean-beta-declarative-pipeline-pipeline-editor/[Back in September 2016 we announced the availability of the Blue Ocean beta]
+and the forthcoming Pipeline Editor. We are happy to announce that you can try
+the Pipeline Editor preview today.
```

Comment:
I suggest phrasing this "preview release", "Pipeline Editor preview today" flows awkwardly IMHO.